### ISSUE TITLE
Accessibility - Headings

### ISSUE THREAD

**Issue by mcmullengreg** [XavierUniversity/V4#14]
In conjunction with #13 we will need to have a better structure for defining headings on banded pages. Content pages have the `h1` defined in the breadcrumbs whereas some banded pages may not include an `h1`. The homepage comes to mind.

As of writing we have (WCAG A standard):
- 36 pages that do not have any headings of any kind
- 235 pages do not have an `h1`
- 1,181 where the heading text is missing

Some of these will also close out WCAG AAA issues. Other AAA issues include: 
- 4,851 pages have improperly structure headings
- 360 Structure Headings
- all A standard issues affect AAA
